src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,22 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
+        return valuesCount > 1
+    }
+
+
+    ProtoValue getValue() {
+        innerProtoProperty.valuesList.first()
```

Comment:
@jbornemann Found an example in /etc/segmentation of a node with resourceType of cq/personalization/components/segmentpage that has no value for the field cq:tags.

Thinking this segment was copied from another and the tag values manually removed in the JCR.